dbaccess/transaction_input.go

```diff
@@ -0,0 +1,7 @@
+package dbaccess
+
+// Outpoint represent an outpoint in a transaction input.
+type Outpoint struct {
```

Comment:
The main part of this file is transaction_input. I've elected to place Outpoint here because TransactionInputs are often searched by outpoint, but it's not interesting anywhere else.

If you disagree lets fight over it